src/sources/kubernetes/mod.rs

```diff
@@ -300,6 +267,93 @@ fn transform_file() -> crate::Result<Box<dyn Transform>> {
     })
 }
 
+/// Contains several regexes that can parse common forms of pod_uid.
+/// On first message, regexes are tryed out one after the other until
+/// first succesfull one has been found. After that that regex will be
+/// always used.
+///
+/// If nothing succeds the message is still passed.
+fn transform_pod_uid() -> crate::Result<ApplicableTransform> {
+    let mut regexes = Vec::new();
+
+    let namespace_regex = r"(?P<pod_namespace>[0-9a-z.\-]*)";
+    let name_regex = r"(?P<pod_name>[0-9a-z.\-]*)";
+    let uid_regex = r"(?P<object_uid>([0-9A-Fa-f]{8}[-][0-9A-Fa-f]{4}[-][0-9A-Fa-f]{4}[-][0-9A-Fa-f]{4}[-][0-9A-Fa-f]{12}|[0-9A-Fa-f]{32}))";
+
+    // Definition of pod_uid has been well defined since Kubernetes 1.14 with https://github.com/kubernetes/kubernetes/pull/74441
+
+    // Minikube 1.15, MicroK8s 1.15,1.14,1.16 , DigitalOcean 1.16 , Google Kubernetes Engine 1.13, 1.14, EKS 1.14
+    // format: namespace_name_UID
+    regexes.push(format!(
+        "^{}_{}_{}$",
+        namespace_regex, name_regex, uid_regex
+    ));
+
+    // EKS 1.13 , AKS 1.13.12, MicroK8s 1.13
```

Comment:
Ah I see the issue here is that EKS is still on 1.13 not 1.14? Is that correct?

Reply:
Yes/Was , and recently it added option to use 1.14. At the moment it is possible to deploy with 1.12, 1.13, and 1.14 versions.